tensilelite/Tensile/Common.py

```diff
@@ -102,7 +102,6 @@
 globalParameters["ExpandRanges"] = True          # expand ranges into exact configs before writing logic file.  False ignores ranges.
 globalParameters["ExitAfterKernelGen"] = False     # Exit after generating kernels
 globalParameters["GenerateSourcesAndExit"] = False # Exit after kernel source generation.
-globalParameters["ShowProgressBar"] = True     # if False and library client already built, then building library client will be skipped when tensile is re-run
```

Comment:
duplicate of line [#L100](https://github.com/ROCm/hipBLASLt/blob/1e4ff1f7cc5fbfa9688c34e1973d9ab389053546/tensilelite/Tensile/Common.py#L100)